src/module.c

```diff
@@ -978,6 +979,26 @@ void RM_ChannelAtPosWithFlags(RedisModuleCtx *ctx, int pos, int flags) {
     res->numkeys++;
 }
 
+/* Returns 1 if name is valid, otherwise returns 0.
+ *
+ * We want to block some chars in module command names that we know can
+ * mess things up.
+ *
+ * There are these characters:
+ * ' ' (space) - issues with old inline protocol.
+ * '\r', '\n' (newline) - can mess up the protocol on acl error replies.
+ * '|' - sub-commands.
+ * '@' - ACL
+ * '=', ':', ',' - info commandstats.
```

Comment:
Not quite convinced we still need to block ':' if we are correctly sanitizing it in the info commands. Of the list, I think this is arguably the character most likely to be used.

Reply:
ok, i agree it's a good compromise.
if we remove `:` then we might not even need to consider this a breaking change,
and as the tests being deleted here indicate, this concern with the `info` format, was already handled anyway.
@enjoy-binbin please make that change